.github/workflows/float8_test.yml

```diff
@@ -53,6 +53,11 @@ jobs:
         uv pip install -r dev-requirements.txt
         uv pip install vllm
         pip install .
-        pytest test/float8 --verbose -s
-        pytest test/integration --verbose -s
-        pytest test/dtypes/test_affine_quantized_float.py --verbose -s
```

Comment:
This removes tests

Reply:
thanks, I had an incorrect assumption about what `test_everything.sh` did - added these back.